platform/source/mbed_retarget.cpp

```diff
@@ -1084,12 +1078,8 @@ extern "C" long PREFIX(_flen)(FILEHANDLE fh)
 #if !defined(COMPONENT_SPE) || !defined(TARGET_TFM)
 
 #if !defined(__MICROLIB)
-#if defined (__ARMCC_VERSION) && (__ARMCC_VERSION >= 6010050)
```

Comment:
@evedon Wasn't `#if defined (__ARMCC_VERSION)` still required here?

Reply:
Well spotted. @rajkan01 we need a patch

Reply:
There was an already existing macro guard `#ifdef __ARMCC_VERSION` at https://github.com/ARMmbed/mbed-os/blob/master/platform/source/mbed_retarget.cpp#L1051, so it is not required.